plugins/@grouparoo/salesforce/src/lib/export/model.ts

```diff
@@ -30,6 +37,19 @@ export function getSalesforceModel(
     recordReferenceMatchField: null,
   };
 
+  let destinationOptionsMapping = destinationOptions;
+  switch (destination) {
+    case SALES_FORCE_DESTINATIONS.object:
+      // explicit default
+      break;
+    case SALES_FORCE_DESTINATIONS.contact:
+      destinationOptionsMapping = getContactsMapping(destinationOptions);
```

Comment:
Instead of calling these here and sharing the defintion so much, what if we passed in the defaults to this method (and not the destination name)? That keeps the dependency chain less tangled.

Reply:
That would remove the methods below and do the merge in this one.

Reply:
Excelent!